.travis.yml

```diff
@@ -1,34 +1,53 @@
 language: python
+sudo: false
+dist: bionic
+
 cache: pip
 
+python:
+  - "3.6"
+  - "3.7"
+
 addons:
   apt:
     packages:
       - sqlite3
+      - fping
       - gdal-bin
+      - openssl
       - libproj-dev
       - libgeos-dev
       - libspatialite-dev
+      - spatialite-bin
+      - libsqlite3-mod-spatialite
 
-python:
-  - "3.6"
+services:
+  - docker
+  - redis-server
 
 branches:
   only:
     - master
 
 before_install:
-  - pip install -U pip wheel setuptools
+  - docker run -d --name influxdb -p 8086:8086 influxdb
+  - docker exec -it influxdb influx -execute 'CREATE DATABASE openwisp2'
+  - pip install -U pip wheel setuptools pyasn1
   - pip install --no-cache-dir -U -r requirements-test.txt
-  - ./runflake8
-  - ./runisort
 
 install:
   - python setup.py -q develop
 
 script:
+  - |
+    openwisp-utils-qa-checks \
+          --migration-path \
+          "./openwisp_monitoring/check/migrations \
+          ./openwisp_monitoring/device/migrations \
+          ./openwisp_monitoring/monitoring/migrations \
+          ./openwisp_monitoring/notifications/migrations" \
+          --migration-module "check device_monitoring monitoring notifications"
   - coverage run --source=openwisp_monitoring runtests.py
```

Comment:
I understood why this is failing, can you add `pip install https://github.com/openwisp/openwisp-utils/tarball/master` temporarily in `before_install` please?